data/starkscan.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "STARKSCAN",
+  "description": "Next-generation StarkNet block explorer.",
+  "short_description": "StarkNet block explorer",
+  "tags": [],
+  "contracts": [],
+  "goerliContracts": [],
+  "audits": [],
+  "dotw": false,
+  "links": {
+    "website": "https://starkscan.co/?utm_source=dappland",
+    "careers": "jobs@aspect.co",
+    "twitter": "https://twitter.com/StarkscanCo",
+    "telegram": "",
+    "discord": "https://discord.gg/Pdt4Wr6gx7",
+    "github": "https://github.com/starkscan",
+    "youtube": "",
+    "medium": "",
+    "mirror": ""
+  },
+  "twitterName": "StarkscanCo",
+  "teamInfo": {
+    "founded": "2022-09-28T00:00:00.000Z",
+    "anonymous": false
+  },
+  "tokens": [],
+  "media": {
+    "logoUrl": "https://pbs.twimg.com/profile_images/1558096338386141185/oT4BGZQr_400x400.jpg",
+    "bannerUrl": "https://pbs.twimg.com/profile_banners/1558095394558644225/1660314194/1500x500",
+    "previewUrl": "https://pbs.twimg.com/profile_banners/1558095394558644225/1660314194/1500x500",
```

Comment:
It looks like this in the homepage. Just checking if you guys have an image that fits better and doesn't crop the text.

<img width="444" alt="Screenshot 2022-10-06 at 16 20 24" src="https://user-images.githubusercontent.com/10310633/194339125-6a8a9252-3161-4e4a-ae05-8cffc89ea35b.png">

Reply:
This is the `previewUrl` i'm guessing? 

Can we try using the `logoUrl in there? See how that looks?

Reply:
looks way better! 
![Screenshot 2022-10-10 at 12 16 36](https://user-images.githubusercontent.com/10310633/194845007-a0ff6e17-cc4c-41c3-855c-1865f55e41fe.png)

Reply:
hello